### PR TITLE
Round robin server and renaming some files for clarity

### PR DESCRIPTION
**Pull request by Marakai**
I've renamed two files for clarity:

- `server.py` is `flask_server.py` because I always got confused with the code dealing with the NGINX server :)
- `runner.py` I renamed to `hostshandler.py` because that's what that Python code does

A new file:

- `google_servers.py` extracts the list of the Google Maps servers you've found

The big change:

In the `server.py`/`flask_server.py` file, I've now introduced an iterator cycle that should round-robin through all the Google servers (in `google_servers.py`). Every call to `tiles()` should return the next server in the list, cycling through forever. Maybe this will prevent some throttling issues (unless Google throttles at a different level of their CDN independent of the target URL).

I still need to test this in detail, but I just took my Flightsim equipment apart, because I'm moving it it to a new place in the house and getting some touch panels and Knobster. 

Please let me know if this works for you!

### CODE REVIEW

File: src/flask_server.py [hunk 1]
```diff
@@ -1,137 +1,156 @@
...
+        if t == '2':
+            tile_y |= mask
+        if t == '3':
+            tile_x |= mask
+            tile_y |= mask
+    return tile_x, tile_y, level_of_detail
+
+
+# TODO: Is this even used anywhere? Pycharm Find Usages can't find anything
+def _calc_brightness(im: any) -> str:
```
Comment (derekhe) [inline src/flask_server.py:154]: It used for print the brightness, not needed any more

File: src/google_servers.py [hunk 2]
```diff
@@ -0,0 +1,8 @@
+GOOGLE_SERVERS = [
+    "khms1.google.com",
```
Comment (derekhe) [inline src/google_servers.py:2]: can use "khm.google.com" and "mt.google.com"
They have load balance

**Review by derekhe — CHANGES_REQUESTED** [review]

File: src/google_servers.py (hunk 2 above)
Comment (Marakai) [reply src/google_servers.py:2]: That's good to know. The question is whether the LB will round-robin or maintain a session connection. In which case there's it the risk of throttling again.

**Comment by Marakai** [comment]
changes made

**Comment by derekhe** [comment]
Hi could you please check latest main branch and your merge request? I removed the built-in cache and seems your commit still have it. If I merge it the built-in cache will comes back.

**Comment by Marakai** [comment]
That's weird, I'm trying to fetch/merge from your upstream main branch and it says I'm up to date. I then created my round-robin branch off that! Will investigate.

**Comment by Marakai** [comment]
For some reason I was not getting your main branch properly rebased on to my PR branch! Had to diff and then update line by line. Sorry about that! It should now have the built-in cache lines removed from `app.py` and `settings.py` while still maintaining my minor changes to those files (`settings.py` only in this PR).

**Comment by Marakai** [comment]
Also: there is a new file `quart_server.py`, but it's not used. I put that in for you, in case you want to experiment with a HTTP/2 capable version of Flask and potentially get rid of NGINX. It's literally a copy of the `server.py`/`flask_server.py` with the necessary changes to imports and use of `async`/`await`. It's just a first step, I don't do enough web API development anymore to know the ins-and-outs of HTTP vs HTTP/2. I mostly do cloud devsecops in Python and Go. :-)

**Comment by derekhe** [comment]
Can I postpone the merge? It because I guess changing to khms and mt may solve the robin issue automatically. Let's test it and wait users' feedback.

**Pull request closed without merging**